### DISCUSSION TITLE
Link style button and Anchor

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#1820] [General] [answered]
Hi there, 

For the Email in Flex project we have a 'Cc' button included when composing an email. Initially we had used the anchor style as the Cc button with the underline always present. The feedback from UX Review panel/design systems was:

> This should be a link-style Button rather than an Anchor, since clicking it performs an in-page action (rather than navigating the user to a different page). _Please see the wide view example_

<img width="1024" alt="Screenshot 2021-08-31 at 15 02 27" src="https://user-images.githubusercontent.com/49287701/131517444-9f45cb4d-e639-40b4-a7c8-3cf3c89a37ee.png">

I have since updated the Cc to be a link style button, however in the slim view of Email in Flex (please see the slim view example) the Cc button appears very large compared to the surrounding UI.

<img width="320" alt="Screenshot 2021-08-31 at 14 58 54" src="https://user-images.githubusercontent.com/49287701/131517480-bb8a494d-b735-4a1e-b18b-212ac1d7ae71.png">

Can we use the Cc link style button here so it's not Bolded? 

Regards,
Porlin

**Comment by TheSisb** [comment, marked as answer]
Hi @Porlin,

I'm a little unclear on the issue, sorry.  

>  in the slim view of Email in Flex (please see the slim view example) the Cc button appears very large compared to the surrounding UI.

> Can we use the Cc link style button here so it's not Bolded?

Is the issue that the word "Email" is bold, that it has too much margin, or something else?

Thanks!

**Reply by Porlin** [reply]
Correct, the Cc is bolded. Are we able to use the Cc so it's just in the regular font weight?

**Reply by nkrantz** [reply]
Hi @Porlin!

Currently, there's no way to change the font weight of the link style button text. In order to do so, we'd have to make a change to the API, so we'd like to see some customer research showing that it's a necessary change. If your customers prefer the lighter font weight, we can consider how to move forward with this change.

Thanks!
